src/Components/VimeoVideo.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef } from 'react';
+
+import { Video } from 'expo-av';
+
+const VimeoVideo = props => {
+  const vimeoUrlSource = `https://player.vimeo.com/video/${props.vimeoId}/config`;
+  const videoElem = useRef(null);
+
+  useEffect(() => {
+    async function fetchVimeoData() {
```

Comment:
Why do you need to define a function and call it just afterwards ?

Reply:
Because it is an `async` function and so return a `Promise`.
And a Hook should return nothing or a cleanup function

Reply:
Maybe we could remove the async/await and just call fetch
Then we could display something while waiting for the video to load ?